tests/js/spec/views/organizationIntegrations/integrationDetailedView.spec.jsx

```diff
@@ -80,28 +79,30 @@ describe('IntegrationDetailedView', function () {
         ],
       ],
     ]);
+  });
 
-    wrapper = mountWithTheme(
+  it('shows integration name, status, and install button', async function () {
+    render(
       <IntegrationDetailedView
         params={{integrationSlug: 'bitbucket', orgId: org.slug}}
         location={{query: {}}}
       />
     );
+    expect(screen.getByText('Bitbucket')).toBeInTheDocument();
+    expect(screen.getByText('Installed')).toBeInTheDocument();
+    expect(screen.getByRole('button', {name: 'Add integration'})).toBeEnabled();
   });
-  it('shows the Integration name and install status', async function () {
-    expect(wrapper.find('Name').props().children).toEqual('Bitbucket');
-    expect(wrapper.find('IntegrationStatus').props().status).toEqual('Installed');
-  });
-  it('shows the Add Installation button', async function () {
-    expect(wrapper.find('AddIntegrationButton').props().disabled).toEqual(false);
-  });
+
   it('view configurations', async function () {
```

Comment:
since we are touching this file. do you think we can remove the `async`? 🙏

Reply:
sgtm